README.md

```diff
@@ -102,6 +102,19 @@ When using GitHub:
   for you, to save `git-delete-merged-branches` some work on the remote side, if you like.
 
 
+# Name clash on "git-delete-merged-branches" with git-extras
+
+Utility collection [git-extras](https://github.com/tj/git-extras) provides a
+[simple shell script named `git-delete-merged-branches`](https://github.com/tj/git-extras/blob/master/bin/git-delete-merged-branches)
+that competes for command `git-delete-merged-branches` when both software packages are installed.
+
+To address this conflict you can:
+
+- a) Invoke `python -m git_delete_merged_branches` instead of `git-delete-merged-branches`
```

Comment:
OnyLlinux python 3 is most of the times python3. Not sure what your target audience is but if it is Linux I would write python3.

Reply:
Alright! :+1: